shared/react-native/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Aug 19 16:09:49 EDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
```

Comment:
is this needed?

Reply:
Yeah, the old gradle didn't work on the latest version of java. I might be the only one who uses gradle though.

Reply:
i think we shouldn't update this. RN docs say we shouldn't update to the latest android studio (etc)

https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#057

> Android tooling has been updated to match newer configuration requirements (SDK 27, gradle 4.4, and support library 27); building with Android plugin 3.2 doesn't work due to the gradle scripts, so please stay on Android Studio 3.1 for now

i don't want to get newer than they are and have to deal with all sorts of junk like this